src/test.rs

```diff
@@ -725,6 +725,22 @@ pub async fn wait_for_nr_cas_under_testbed(nr: usize) -> bool {
     false
 }
 
+pub async fn wait_for_nr_cas_under_publication_server(publishers_expected: usize) {
+    let mut publishers_found = list_publishers().await.publishers().len();
+    for _ in 0..300 {
+        if publishers_expected == publishers_found {
```

Comment:
Personally I'd swap the order of these comparison arguments as it reads to me as if `publishers_expected` should change to match `publishers_found`.

Reply:
fixed